docs/api/browser-window.md

```diff
@@ -391,6 +391,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       contain the layout of the document—without requiring scrolling. Enabling
       this will cause the `preferred-size-changed` event to be emitted on the
       `WebContents` when the preferred size changes. Default is `false`.
+    * `mainFrameName` String (optional) - The name assigned to the top-level
+      frame of the `WebContents` instance.
```

Comment:
Maybe mentioning `<a target="foo">` and `window.open('', 'bar')` as examples here, "frame name" is not a very well known term.

Reply:
Do you think it'd be okay to link the MDN article on this? https://developer.mozilla.org/en-US/docs/Web/API/Window/name

> The [name](https://developer.mozilla.org/en-US/docs/Web/API/Window/name) assigned to the top-level frame of the `WebContents` instance.

Reply:
Yeah it should be enough.